cdap-app-fabric/src/main/java/co/cask/cdap/metadata/MetadataSubscriberService.java

```diff
@@ -286,4 +296,48 @@ public void processMessage(MetadataMessage message) {
       }
     }
   }
+
+  private class MetadataOperationProcessor implements MetadataMessageProcessor {
```

Comment:
Please add a javadoc for this class to tell what message type it is responsible for.

Reply:
sure.